.github/workflows/tests.yml

```diff
@@ -64,6 +64,8 @@ jobs:
         include:
           - build_type: system
             target: chat
+          - build_type: production
+            target: core
```

Comment:
we should probably start splitting this mega-workflow file into separate backend/frontend/system workflows… 😅 

like, does this job need postgres and redis?

---

also, do we want it to use our runners or gh's? (gh would probably be fine?)

Reply:
> also, do we want it to use our runners or gh's? (gh would probably be fine?)

Depends on whether we can run this with a single core or not. `ubuntu-latest` is free for public repos so we can use it as much as we want in this repo. One downside I discovered recently is that we are getting charged for `ubuntu-latest` in our private mirror and I'm still thinking about how to resolve that. We have limited self hosted runners so I don't really want to increase the number of runners we require per build.